python/xorbits/_mars/lib/filesystem/fsspec_adapter.py

```diff
@@ -35,6 +35,8 @@ def cat(self, path: path_type) -> bytes:
     def ls(self, path: path_type) -> List[path_type]:
         entries = []
         for entry in self._fs.ls(self._normalize_path(path), detail=False):
+            if entry.strip("/") in path.strip("/"):
```

Comment:
Why not use equal?

Reply:
because the head of `path` will be `s3:\\`, which can not be removed by `strip`